functions.php

```diff
@@ -4,6 +4,12 @@
  * Additional code for the child theme goes in here.
  */
 
+add_action( 'wp_enqueue_scripts', 'gpnz_enqueue_script' );
+function gpnz_enqueue_script() {
```

Comment:
Enqueueing function looks good

It would a bit of improvement if you enqueue this only posts/pages that have enblocks.

```
if( is_a( $post, 'WP_Post' ) && has_shortcode( $post->post_content, 'shortcake_enblock') ) {

}
```

https://codex.wordpress.org/Function_Reference/has_shortcode#Simple_Example_2

Reply:
Thanks I have added this & removed the JS check